deployment/migrations/scripts/0003-convert-key-to-pkcs8-der.py

```diff
@@ -0,0 +1,78 @@
+"""
+This migration converts the PEM private key file to PKCS8 DER for compatibility
+with the new Aleph.im P2P service. The Rust implementation of libp2p can only load
+RSA keys in that format.
+"""
+
+
+import logging
+import os
+from pathlib import Path
+from typing import Optional
+
+import yaml
+from Crypto.PublicKey import RSA
+from p2pclient.libp2p_stubs.crypto.rsa import RSAPrivateKey
+
+from aleph.exceptions import InvalidKeyDirException
+
+LOGGER = logging.getLogger(os.path.basename(__file__))
+
+
+PKCS8_DER_KEY_FILE = "node-secret.pkcs8.der"
```

Comment:
Should this go in a config file ?

Reply:
The generation code only uses this name, so unless some users want to let's use a default name for the moment.